packages/ses/src/scope-handler.js

```diff
@@ -10,12 +13,6 @@ import { assert } from './error/assert.js';
 
 const { details: d, quote: q } = assert;
 
-// The original unsafe untamed eval function, which must not escape.
-// Sample at module initialization time, which is before lockdown can
-// repair it.  Use it only to build powerless abstractions.
-// eslint-disable-next-line no-eval
-const FERAL_EVAL = eval;
```

Comment:
I’ll note that there were duplicates of this block, complete with matching comments. They are now in `commons.js`.